runtime.md

```diff
@@ -34,22 +34,23 @@ This is provided so that consumers can find the container's configuration and ro
 ```
 
 ## Lifecycle
-
-### Create
-
-Creates the container: file system, namespaces, cgroups, capabilities.
-
-### Start (process)
-
-Runs a process in a container.
-Can be invoked several times.
-
-### Stop (process)
-
-Not sure we need that from runc cli.
-Process is killed from the outside.
-
-This event needs to be captured by runc to run onstop event handlers.
+The lifecycle describes the timeline of events that happen from when a container is created to when it ceases to exist.
+
+1. OCI compliant runtime is invoked by passing the bundle path as argument.
+2. The container's runtime environment is created.
+3. The container's state is written to the filesystem.
+4. The prestart hooks are invoked by the runtime.
```

Comment:
Why are we using the keyword `runtime` here? Are we precluding the Spec from ever being used by humans :smirk: ?